Formula/dssim.rb

```diff
@@ -13,12 +13,37 @@ class Dssim < Formula
     sha256 "51756e74240d03c87a79ff0e14e494d249701d4069ea336b529b47a179c469d3" => :mavericks
   end
 
+  option "with-library", "Build library (require `python3` and `ninja`)"
+
+  if build.with? "library"
+    resource "meson-0.27.0" do
```

Comment:
we have a meson formula

Reply:
Building `dssim-1.3.2` needs just `meson-0.27.0`.
`meson-0.28.0` and newer was removed `pkbconfig_gen()` function which was used in "meson.build" of `dssim-1.3.2`.

Reply:
They'll need to fix that then.

Reply:
I will try offical patch as soon as possible.

Reply:
thanks :)